src/components/PatronBlock/PatronBlockModal.js

```diff
@@ -21,6 +28,8 @@ const PatronBlockModal = ({ open, onClose, patronBlocks, viewUserPath }) => {
     );
   });
 
+  const canBeOverridden = stripes.hasPerm('ui-users.overridePatronBlock');
```

Comment:
I think you can replace `hasPerm` with `IfPermission`. That way you can avoid passing stripes as a prop to `PatronBlockModal `.

https://github.com/folio-org/stripes-core/blob/629c92769d62aacf41dc5af830332a0de127714e/src/components/IfPermission/readme.md

Reply:
Thanks for review. Looks better now, glad to get rid of the extra prop.